process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
 title: DocuSign connector
 ---
 
-The DocuSign connector is used to send documents via email to be digitally signed. The process flow waits for a document to be signed before continuing with the process. The signed document is saved to the Content Services repository. Multiple signers is supported, which means more than one name, email address and signature, can be used.
+The DocuSign connector provides a standard mechanism in Alfresco Process Automation to send documents for digital signing in
+[DocuSign](https://www.docusign.com){:target="_blank"}.
 
-> **Important**: The DocuSign connector requires a [DocuSign](https://www.docusign.com/){:target="_blank"} account to handle document signing. This account is separate to the Alfresco hosted environment and should be created and managed by customers.
+## Installation
 
-The DocuSign connector is displayed on the process diagram as a pen.
+The DocuSign connector is a Spring boot application that is included as a separate service to the Process Automation deployment.
 
-## Sign document
+## Configuration
```

Comment:
I think this part of the project's README documentation is intended for a technical user. I wouldn't include it here. I would leave the section as it was before, respecting the format with the rest of the connectors as well:  https://github.com/Alfresco/docs-alfresco/pull/622/files#diff-602426ee5a6a65a534806a831d0df2977a91558480b6e93940165d5ec68b1b8cL66

Reply:
Done